### ISSUE TITLE
obsidian-show-dotfiles is not available in Obsidian plugins

### ISSUE THREAD

**Issue by mwoz123** [polyipseity/obsidian-show-dotfiles#1]
I'm not able to install/find the plugin from `Obsidian community plugins`.

Quote: 
> [community plugin]: https://obsidian.md/plugins?id=show-dotfiles
> 1. Install the [plugin][community plugin] from community plugins directly.

source: https://github.com/polyipseity/obsidian-show-dotfiles/blob/main/README.md

no results in comunity plugins :(

**Comment by ioneliabuzatu** [comment]
true. 
current solution: i did it by following the manual installation as explained in the readme

**Comment by polyipseity** [comment]
Yes, it is not there. I have struck-through the said instructions for now.
The manual method and the BRAT method should work.
(2d6d8c828fabf23e51b810d1e04d9530e6e33d13)

**Comment by mwoz123** [comment]
Just for others to fyi it's not in the shop cause it might break some functionalies especially Obsidian Sync

https://github.com/obsidianmd/obsidian-releases/pull/1959
